xbmc/games/windows/GUIWindowGames.cpp

```diff
@@ -265,6 +265,14 @@ bool CGUIWindowGames::GetDirectory(const std::string &strDirectory, CFileItemLis
   if (!content.empty())
     items.SetContent(content);
 
+  // Ensure files have a game info tag so that they're recognized as games
+  for (auto it = items.begin(); it != items.end(); ++it)
```

Comment:
use a range based for loop

Reply:
`items` is a `CFileItemList`, I tried to figure out how to range-based loop over it but couldn't. Do you know anywhere this is done?

Reply:
I don't know if it is already done anywhere else, but if you look at
https://github.com/xbmc/xbmc/blob/f4d6b4022fc3122471f0d39aeb58abafc9412a10/xbmc/FileItem.h#L761-L766
and
https://github.com/xbmc/xbmc/blob/f4d6b4022fc3122471f0d39aeb58abafc9412a10/xbmc/FileItem.h#L598
you'll come to the following result.
```cpp
  for (const CFileItemPtr& item : items)
  {
    if (!item->m_bIsFolder)
      item->GetGameInfoTag();
  }

```

Reply:
updated